src/registry_schemas/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.5.0'  # pylint: disable=invalid-name
+__version__ = '2.5.1'  # pylint: disable=invalid-name
```

Comment:
Overall we're going with 2.6.0 but since this is a different project we can up-version it separately. What do you think?

Reply:
Yeah my thought was up-versioning this separately and since it's a minor update that doesn't break anything using 2.5.0 I figured this was good. Will change it to 2.6.0 if that's more what everyone else is thinking tho!

Reply:
It's not the same version, if we want everything to be tagged with the same version numbers, then we can put them all back into 1 repo